src/ut/alarm_scheduler_test.cpp

```diff
@@ -472,9 +473,9 @@ TEST_F(AlarmSchedulerTest, AlarmFlicker)
   }
 
   cwtest_advance_time_ms(AlarmScheduler::ALARM_REDUCED_DELAY);
-  pthread_mutex_lock(&(_alarm_scheduler->_lock));
+  pthread_mutex_lock(&_alarm_scheduler->_lock);
```

Comment:
Could use `_lock` here (and the other similar places) I think. Delving into the internals of a class to grab a lock is a bit icky, but locking on something you own, and passed a reference to the class under test is cleaner.

Reply:
The indirection through `_alarm_scheduler` isn't part of my change, and (since I've already live-tested) I want to draw the line here and leave the code as-is.  OK?

Reply:
Done.